### PR TITLE
feat: Add limits for voting delay and period

### PR DESCRIPTION
**Pull request by ChaituVR**
## Changes:
- Refer https://discord.com/channels/955773041898573854/1030772407226601522/1210204913381023794
- Add one month limit to voting delay
- Add 6 month limit for voting period 
- Add custom errors using `errorMessage` so error will be `"Delay must be less than 30 days"` instead of `Must be < 2592000`

## How to test:
- Run `yarn test` 
- all tests should pass
- Change values in `test/examples/space.json` to something more than limits
- Notice the new error message

### CODE REVIEW

File: test/examples/space.json [hunk 1]
```diff
@@ -28,5 +28,9 @@
   "validation": {
     "name": "basic",
     "params": {}
+  },
+  "voting":{
+    "delay": 2592000,
+    "period": 31536000
```
Comment (ChaituVR) [inline test/examples/space.json:34]:
```suggestion
    "period": 15811200
```

File: src/schemas/space.json [hunk 2]
```diff
@@ -282,11 +282,19 @@
...
+              }
             },
             "period": {
               "type": "integer",
-              "minimum": 0
+              "minimum": 0,
+              "maximum": 31536000,
+              "errorMessage": {
+                "maximum": "Delay must be less than 1 year"
+              }
```
Comment (ChaituVR) [inline src/schemas/space.json:297]:
```suggestion
              "maximum": 15811200,
              "errorMessage": {
                "maximum": "Delay must be less than 183 days"
              }
```

**Review by wa0x6e — APPROVED** [review]
tAck

File: src/schemas/space.json [hunk 3]
```diff
@@ -282,11 +282,19 @@
...
+                "maximum": "Delay must be less than 30 days"
+              }
             },
             "period": {
               "type": "integer",
-              "minimum": 0
+              "minimum": 0,
+              "maximum": 15811200,
+              "errorMessage": {
+                "maximum": "Delay must be less than 183 days"
```
Comment (wa0x6e) [inline src/schemas/space.json:296]: Maybe use 180 days, which is a more widely accepted value for half year

Comment (ChaituVR) [inline src/schemas/space.json:296]:
```suggestion
              "maximum": 15552000,
              "errorMessage": {
                "maximum": "Delay must be less than 180 days"
```

File: test/examples/space.json [hunk 4]
```diff
@@ -28,5 +28,9 @@
   "validation": {
     "name": "basic",
     "params": {}
+  },
+  "voting":{
+    "delay": 2592000,
+    "period": 15811200
```
Comment (ChaituVR) [inline test/examples/space.json:34]:
```suggestion
    "period": 15552000
```

**Pull request merged**